restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/UsageRecordsTest.java

```diff
@@ -0,0 +1,273 @@
+        String price_unit = firstRecord.get("price_unit").getAsString();
+        String uri = firstRecord.get("uri").getAsString();
+
+        assertTrue(category.equals("calls"));
+        assertTrue(description.equals("Total Calls"));
+        assertTrue(startDate.equals("2016-01-01"));
+        assertTrue(endDate.equals("2017-12-31"));
+        assertTrue(count.equals("36"));
+        assertTrue(usage.equals("360"));
+        assertTrue(price.equals("180.0"));
+        assertTrue(count_unit.equals("calls"));
+        assertTrue(usage_unit.equals("minutes"));
+        assertTrue(price_unit.equals("USD"));
+        //FIXME:start date should be start of epoch?
+        assertTrue(uri.equals("/2012-04-24/Accounts/ACae6e420f425248d6a26948c17a9e2acf/Usage/Records/AllTime.json?Category=Calls&EndDate=2017-02-03&StartDate=2016-01-01"));
+
+        //TODO: test other categories
+    }
+
+    @Test
+    public void getUsageRecordsToday() {
+        JsonElement firstPage = RestcommUsageRecordsTool.getInstance().getUsageRecordsToday(deploymentUrl.toString(),
+                adminAccountSid, adminAuthToken, "", true);
+        logger.info(firstPage);
+    }
+
+    @Test
+    public void getUsageRecordsYesterday() {
+        JsonElement firstPage = RestcommUsageRecordsTool.getInstance().getUsageRecordsYesterday(deploymentUrl.toString(),
+                adminAccountSid, adminAuthToken, "", true);
+        logger.info(firstPage);
+    }
+
+    @Test
+    public void getUsageRecordsThisMonth() {
+        JsonElement firstPage = RestcommUsageRecordsTool.getInstance().getUsageRecordsThisMonth(deploymentUrl.toString(),
+                adminAccountSid, adminAuthToken, "", true);
+        logger.info(firstPage);
+    }
+
+    @Test
+    public void getUsageRecordsLastMonth() {
+        JsonElement firstPage = RestcommUsageRecordsTool.getInstance().getUsageRecordsLastMonth(deploymentUrl.toString(),
+                adminAccountSid, adminAuthToken, "", true);
+        logger.info(firstPage);
+    }
+
+    @Deployment(name = "UsageRecordsTest", managed = true, testable = false)
+    public static WebArchive createWebArchiveNoGw() {
+        WebArchive archive = ShrinkWrap.create(WebArchive.class, "restcomm.war");
+        final WebArchive restcommArchive = ShrinkWrapMaven.resolver()
+                .resolve("org.restcomm:restcomm-connect.application:war:" + version).withoutTransitivity()
+                .asSingle(WebArchive.class);
+        archive = archive.merge(restcommArchive);
+        archive.delete("/WEB-INF/sip.xml");
+        archive.delete("/WEB-INF/conf/restcomm.xml");
+        archive.delete("/WEB-INF/data/hsql/restcomm.script");
+        // archive.delete("/WEB-INF/data/hsql/restcomm.properties");
+        archive.addAsWebInfResource("sip.xml");
+        archive.addAsWebInfResource("restcomm.xml", "conf/restcomm.xml");
+        archive.addAsWebInfResource("restcomm_with_Data_UsageRecords.script", "data/hsql/restcomm.script");
```

Comment:
@otsakir i updated the commit, i remember adding the `restcomm_with_Data_UsageRecords.script` script, i guess i was wrong!
You are right about the params, but i did not intend to change/use `restcomm_with_Data.script`.